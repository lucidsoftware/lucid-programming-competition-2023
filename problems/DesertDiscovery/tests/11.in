1
Sp6gkNEF9IFBXU9nD00jfNvKnUpNQkPcLY
3
AtEOqdY1wOPCrKXW2RFnSQPuEauruHrdi2
AtjqvpG2MDeLNQzdPLH7KzCN2TRMi1buY1
Ats1ND8U5nSakLxlCJVleH89S7HupcItZ1
5
2
Ats1ND8U5nSakLxlCJVleH89S7HupcItZ1
AtEOqdY1wOPCrKXW2RFnSQPuEauruHrdi2
2
Ats1ND8U5nSakLxlCJVleH89S7HupcItZ1
AtjqvpG2MDeLNQzdPLH7KzCN2TRMi1buY1
3
AtjqvpG2MDeLNQzdPLH7KzCN2TRMi1buY1
Ats1ND8U5nSakLxlCJVleH89S7HupcItZ1
AtEOqdY1wOPCrKXW2RFnSQPuEauruHrdi2
3
AtjqvpG2MDeLNQzdPLH7KzCN2TRMi1buY1
AtEOqdY1wOPCrKXW2RFnSQPuEauruHrdi2
Ats1ND8U5nSakLxlCJVleH89S7HupcItZ1
3
AtEOqdY1wOPCrKXW2RFnSQPuEauruHrdi2
Ats1ND8U5nSakLxlCJVleH89S7HupcItZ1
AtjqvpG2MDeLNQzdPLH7KzCN2TRMi1buY1
