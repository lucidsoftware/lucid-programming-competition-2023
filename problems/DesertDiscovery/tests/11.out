Sp6gkNEF9IFBXU9nD00jfNvKnUpNQkPcLY
Sp6gkNEF9IFBXU9nD00jfNvKnUpNQkPcLY
Sp6gkNEF9IFBXU9nD00jfNvKnUpNQkPcLY
Sp6gkNEF9IFBXU9nD00jfNvKnUpNQkPcLY
Sp6gkNEF9IFBXU9nD00jfNvKnUpNQkPcLY
